BeagleBone_black/src/navigation/navigation.cpp

```diff
@@ -384,27 +382,47 @@ void Navigation::stripeCounterUpdate(array<StripeCounter, Sensors::kNumKeyence>
       displacement_[0], displacement_[1], displacement_[2]);
   // TODO(Brano): Update displacement and velocity
 
-  // TODO(Brano): Change this once 2nd Keyence is added
-  if (!sc[0].operational) {
+  int num_operational = 0;
+  for (unsigned int i = 0; i < data::Sensors::kNumKeyence; ++i) {
+    if (scs[i].operational) {
+      // This operational check could be redundant since keyence is always operational (for now)
+      ++num_operational;
+    }
+  }
+
+  if (num_operational < data::Sensors::kNumKeyence) {
```

Comment:
I think we can do fine with just one keyence since the algorithm doesn't care if the two sensors are out of sync